src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java

```diff
@@ -178,7 +179,7 @@ public boolean hasSdoBeenDrawn() {
 
     @Override
     public boolean isBeforeHearing() {
-        return (isBeforeSmallClaimHearing() || isBeforeFastTrackHearing()) || noHearingScheduled();
+        return isBeforeSmallClaimHearing() || isBeforeFastTrackHearing();
```

Comment:
Sorry Kenneth, are you sure the fields for small claim hearing and fast track hearing have read access for lip defendant?

Reply:
Added related field authorisation in the ccd side